master/reference/repo-structure.md

```diff
@@ -1,69 +1,63 @@
-respositories are listed below.
-
-## Calico Libraries
-
- - [calico](https://github.com/projectcalico/calico): This repo contains the
-   implementation of Felix which is the heart of Calico networking.  Felix
-   interfaces with the Linux kernel to configure routes and ACLs
-   that control network policy connectivity. In `calico/node`, Felix runs as a
-   separate process.  It is installed directly into the `calico/node` image.
-
- - [libcalico](https://github.com/projectcalico/libcalico): Contains a variety
-   of helper methods and classes for manipulating the Calico data in an etcd
-   datastore, IP address management and useful namespace utilities to
-   manipulating container interfaces.  The libcalico library is used by
-   `calicoctl`.  It is imported as a standard Python library into the
-   `calicoctl` Python codebase.  libcalico also contains the `calico/test` Docker
-   image used for running both UTs and STs testing Calico with Docker.
-
-## Calico Orchestrator Integration Plugins
+## Core Components
+
+### [Felix](https://github.com/projectcalico/felix)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `felix` | Binary | Felix interfaces with the Linux kernel to configure routes and ACLs that control network policy connectivity. |
+
+
+### [Calico-Containers](https://github.com/projectcalico/calico-containers)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| `calico/node`| Docker Image |  The Docker image used to run Felix and the BIRD BGP agent (for distributing routes between hosts).  See [`calico_node` directory]({{site.baseurl}}/{{page.version}}/reference/architecture) for details. |
+| `calicoctl`| Binary | The command line tool for interacting with the Calico API.  See [`calicoctl` directory]({{site.baseurl}}/{{page.version}}/reference/calicoctl) for implementation of calicoctl. |
+
+## Libraries
+
+### [libcalico](https://github.com/projectcalico/libcalico)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico | pypi library | Contains a variety of helper methods and classes for manipulating the Calico data in an etcd datastore, IP address management and useful namespace utilities to manipulating container interfaces. |
+| `calico/test` | Docker Image | Contains useful shared testing framework code and dependencies for running unit and system tests of Calico in Docker. |
+| `calico/build` | Docker Image | Build image which includes libcalico and the necessary python tooling to produce binaries from python source code. |
+
+## [libcalico-go](https://github.com/projectcalico/libcalico-go)
+
+| Artifact | Type | Description |
+|---------|-------|-----------|
+| libcalico-go | golang library | Contains a variety of helper methods and classes for managing Calico's data, performing IP address management, and providing useful namespace utilities to manipulate container interfaces. |
```

Comment:
> providing useful namespace utilities to manipulate container interfaces.

I don't think this is true in the Go version.

> for managing Calico's data,

This should be worded as "for interacting with the Calico API"